### DISCUSSION TITLE
I have a warning in Meta ads that the SDK is not up to date

### DISCUSSION THREAD

**Discussion by Samox** [thebergamo/react-native-fbsdk-next#554] [Q&A]
Facebook released 17.1 in beginning of September https://github.com/facebook/facebook-ios-sdk/releases/tag/v17.1.0 
In my app I currently have the 17.0.3

Meta always shows weird and obsoletes warnings, so I don't know if it is related to the version, or just something else I did wrong 🙃
Has anyone the same problem ?

**Comment by commbits** [comment]
Same here. Apparently react-native-fbsdk-next 13.0.0 comes with Facebook SDK 17.0.3, but Meta recommends 17.1. 
I'm not sure if the note on Meta is just a warning or whether it has other side-effects.

Anyone here knows about any plans to update react-native-fbsdk-next with Facebook SDK 17.1?

**Comment by mikehardy** [comment]
This is being tracked as an issue over on #555 - best not to split discussion
I've offered to do the update over there but haven't heard back?
